pennylane_qiskit/converter.py

```diff
@@ -1042,3 +1046,94 @@ def _expr_eval_clvals(clbits, clvals, expr_func, bitwise=False):
+        readout_error (bool): include readout errors in the converted noise model. Default is ``True``
+        kraus_shape (bool): use shape of the Kraus operators to display ``qml.QubitChannel``
+            instead of the complete list of matrices. Default is ``True``
+        decimals (int): number of decimal places to round the Kraus matrices. Default is ``10``
+
+    Returns:
+        pennylane.NoiseModel: An equivalent noise model constructed in PennyLane
+
+    Raises:
+        ValueError: When an encountered quantum error cannoted be converted.
+
+    .. note::
+
+        Currently, PennyLane noise models does not support readout errors, so those will be skipped during conversion.
+
+    **Example**
+
+    Consider the following noise model constructed in Qiskit:
+
+    .. code-block:: python
+
+        >>> import qiskit.providers.aer.noise as noise
+        >>> error_1 = noise.depolarizing_error(0.001, 1) # 1-qubit noise
+        >>> error_2 = noise.depolarizing_error(0.01, 2) # 2-qubit noise
+        >>> noise_model = noise.NoiseModel()
+        >>> noise_model.add_all_qubit_quantum_error(error_1, ['rz', 'ry'])
+        >>> noise_model.add_all_qubit_quantum_error(error_2, ['cx'])
+        >>> load_noise_model(noise_model)
+        NoiseModel({
+            OpIn(['RZ', 'RY']): QubitChannel(Klist=Tensor(4, 4, 4))
+            OpIn(['CNOT']): QubitChannel(Klist=Tensor(16, 4, 4))
+        })
+
+    Equivalently, in PennyLane this will be:
+
+    .. code-block:: python
+
+        import numpy as np
+        import pennylane as qml
+        import itertools as it
+        import functools as ft
+
+        pauli_mats1 = list(map(qml.matrix, [qml.I(0), qml.X(0), qml.Y(0), qml.Z(0)]))
+        pauli_mats2 = list(
+            ft.reduce(np.kron, prod, 1.0) for prod in it.product(pauli_mats1, repeat=2)
+        )
+
+        pauli_prob1 = np.sqrt(error_1.probabilities)
+        pauli_prob2 = np.sqrt(error_2.probabilities)
+
+        kraus_ops1 = [prob * kraus_op for prob, kraus_op in zip(pauli_prob1, pauli_mats1)]
+        kraus_ops2 = [prob * kraus_op for prob, kraus_op in zip(pauli_prob2, pauli_mats2)]
+
+        c0 = qml.noise.op_eq(qml.RZ) | qml.noise.op_eq(qml.RY)
+        c1 = qml.noise.op_eq(qml.CNOT)
+
+        n0 = qml.noise.partial_wires(qml.QubitChanel(kraus_ops1))
+        n1 = qml.noise.partial_wires(qml.QubitChanel(kraus_ops2))
+
+        equivalent_pl_noise_model = qml.NoiseModel({c0: n0, c1: n1})
+    """
+    qerror_dmap, _ = _build_noise_model_map(noise_model, **kwargs)
+    model_map = {}
+    for error, wires_map in qerror_dmap.items():
+        conditions = []
+        for wires, operations in wires_map.items():
+            cond = qml.noise.op_in(operations)
+            if wires != AnyWires:
+                cond &= WiresIn(wires)
+            conditions.append(cond)
+        fcond = reduce(lambda cond1, cond2: cond1 | cond2, conditions)
```

Comment:
Question: Rather than do an array of conditions and ORing across all elements, can we just set `fcond = False` at the beginning and simply do 
```
if qml.noise.op_in(operations):
    fcond = True
```
Effectively I think it's the same unless you need the `conditions` array for something else.